packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -156,8 +157,8 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
     onRenderAvatar,
     showMessageStatus,
     messageStatus,
-    /* @conditional-compile-remove(file-sharing) */
     fileDownloadHandler,
```

Comment:
just wondering if @conditional-compile-remove(teams-inline-images-and-file-sharing) missed here?

Reply:
onInlineImageClicked is just used in the case of image gallery so this should be ok